packages/db/prisma/schema.prisma

```diff
@@ -69,12 +69,15 @@ model BlobDataStorageReference {
 }
 
 model Block {
-  hash       String   @id
-  number     Int      @unique
-  timestamp  DateTime @unique
-  slot       Int      @unique
-  insertedAt DateTime @default(now())
-  updatedAt  DateTime @default(now())
+  hash                  String   @id
+  number                Int      @unique
+  timestamp             DateTime @unique
+  slot                  Int      @unique
+  blobGasUsed           BigInt
+  excessBlobGas         BigInt
+  blobAsCalldataGasUsed BigInt
```

Comment:
I was a bit confused with having the same naming for this field for both the block and tx. I'm not against it, though I wonder if something like `blobAsCalldataSum` could be more explicit 🤔

Reply:
I'd prefer to keep consistency with the field `blobGasUsed` which also contains a sum metric 🤔

Reply:
`blobAsCalldata` still doesn't convince me entirely so maybe we can find a better name for it